example-local/README.md

```diff
@@ -28,3 +31,6 @@ or
 
 **Username** : `user@conduktor.io`    
 **Password** : `user` 
+
+## Requirements
+* Conduktor platform version `>0.11.1`
```

Comment:
Since we're not communicating versions to customers yet, I think it's going to mislead more than this will help